### DISCUSSION TITLE
a lot of "not found", "failed" and "could not be found", while cmake generates a cpp solution

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1078] [Q&A] [answered]
a lot of "not found", "failed" and "could not be found", while cmake generates a cpp solution
![image](https://user-images.githubusercontent.com/120378517/221360642-8b593414-012b-4680-a12c-7436467c0d45.png)

after that, the standard project does not compile for me. I don't know what the problem is, past projects were created without errors and everything was fine

**Comment by halx99** [comment]
which target do you build for?

**Reply by 10100010010001001010** [reply]
write more verbatim, please. I can read English, but I dont understand the meaning of this question

**Reply by aismann** [reply]
> write more verbatim, please. I can read English, but I dont understand the meaning of this question

Show use the whole output plz (at least the first ~10 lines)

**Reply by 10100010010001001010** [reply]
![image](https://user-images.githubusercontent.com/120378517/221372530-1667eaa5-578a-4796-9e2a-1979076ac55f.png)
**the bottom line is the beginning of the previous screenshot**

**Comment by halx99** [comment]
For windows x86, please check the latest github ci:
https://github.com/axmolengine/axmol/actions/runs/4270724934/jobs/7434703791

The output is normally:

![Screenshot_2023-02-26-04-32-14-052-edit_com android chrome](https://user-images.githubusercontent.com/6977319/221378463-cabd791c-2f49-4eaa-b644-aed6952d99c4.jpg)

plz provide your complile error log.

**Reply by 10100010010001001010** [reply]
sorry, but I dont have an existing project anymore, and I cant create a new solution cause of this issue. I only have cmake error log but half is in russian. Can this problem be caused by changing the processor from intel to ryzen and returning to intel again? Its all started right after that

**Comment by halx99** [comment]
Plz upload complete CMakeError.log file, I can't see any error from your screenshot

**Reply by 10100010010001001010** [reply]
[CMakeError.log](https://github.com/axmolengine/axmol/files/10833271/CMakeError.log)

**Reply by halx99** [reply]
Sorry, my mistake, need full output for generate commands: `cmake -B build`

**Reply by 10100010010001001010** [reply]
[CMakeOutput.log](https://github.com/axmolengine/axmol/files/10833333/CMakeOutput.log)
this one?

**Reply by halx99** [reply]
No, I means the console output when you execute generate command: `cmake -B build`, please delete `build/CMakeCache.txt` before run it.

**Reply by 10100010010001001010** [reply]
![1](https://user-images.githubusercontent.com/120378517/221402642-73255d8b-be17-4b58-9a8d-e3709e67ab23.png)
![2](https://user-images.githubusercontent.com/120378517/221402707-2b567574-e1fb-4f66-896c-53aee9f9d565.png)
![3](https://user-images.githubusercontent.com/120378517/221402645-b2fc3a08-7d22-4f65-a85f-fa854e6efd1a.png)

**Reply by halx99** [reply]
It seems you not run `python setup.py`, it's fail with `NUGET.exe not found`

**Comment by halx99** [comment]
Try delete `build/CMakeCache.txt` and regenerate .sln with cmake again.

**Reply by 10100010010001001010** [reply]
it did not help

**Comment by halx99** [comment, marked as answer]
Once you move engine folder, you needs run `python setup.py` in axmol root directory again

**Reply by 10100010010001001010** [reply]
But I dont remember move the folder, and also tried to run setup.py before posting here and I got this) 
![image](https://user-images.githubusercontent.com/120378517/221403617-86efc1c6-090e-4a9c-bb4a-66d0170c462c.png)


after a few repetitions of this message, the console simply closes. I completely forgot to write about it, sorry

**Reply by 10100010010001001010** [reply]
I will try to download it manually

**Reply by halx99** [reply]
so, download nuget fail on your computer.

**Reply by halx99** [reply]
> I will try to download it manually

ok
